examples/embed_ipython.py

```diff
@@ -0,0 +1,18 @@
+"""
+Start napari and land directly in an embedded ipython console with qt event loop
```

Comment:
Could you maybe add some additional detail on why someone would choose to do this as opposed to use the built-in console with `viewer.update_console(locals())`. You can link to this example too https://github.com/napari/napari/blob/master/examples/update_console.py, which we might want to simplify too. I just want to make sure that a beginner or someone coming new to this isn't confused